src/app/pages/PerpetualPage/components/ToastsWatcher/index.tsx

```diff
@@ -66,12 +67,10 @@ export const ToastsWatcher: React.FC = () => {
           );
         }
       }
-      setToastedTransactions(transactions => ({
-        ...transactions,
-        [transaction.transactionHash]: true,
-      }));
+
+      dispatch(actions.pushToastedTransactions(transaction.transactionHash));
```

Comment:
Sorry I don't want to be pedantic, :face_exhaling: **but**
```suggestion
      dispatch(actions.pushToastedTransaction(transaction.transactionHash));
```

Reply:
Sorry, I was probably blind :D